clang/runtime/dpct-rt/include/dpct/blas_gemm_utils.hpp

```diff
@@ -430,6 +448,13 @@ inline void int2float(queue_ptr q_ptr, void *int_ptr, bool is_host_ptr,
     });
   }
 }
+
+inline void multiply(queue_ptr q_ptr, float *result, float *a, float *b) {
```

Comment:
This function should be a template function.
We also need a wrapper to do type dispatch.